src/ui/actions/timeline.ts

```diff
@@ -180,7 +180,7 @@ export function setTimelineState(state: Partial<TimelineState>): SetTimelineStat
   return { type: "set_timeline_state", state };
 }
 
-export function setTimelineToTime(time: number | null, updateGraphics = true): UIThunkAction {
+export function setTimelineToTime(time: number | null, updateGraphics = false): UIThunkAction {
```

Comment:
That was a drive-by cleanup because I thought this default was a holdover from when we repainted whenever the user hovered over an item in the UI that had an associated time and it never did anything useful anymore. Looking at it again I realized that those hover features can still be enabled, so I'll change it back.